l10nfr.properties

```diff
@@ -33,7 +33,19 @@ defaults.mavenArtifactId=sonar-l10n-fr-plugin
 9.9.0.changelogUrl=https://github.com/jycr/sonar-l10n-fr/releases/tag/9.9.0
 
 9.9.1.description=Fixes misleading translations and uses right apostrophe quotation mark
-9.9.1.sqVersions=[9.9,LATEST]
+9.9.1.sqVersions=[9.9,9.9.*]
```

Comment:
Since SQ10.0 is out, this range needs to end in 10.0. Otherwise SQ10.0 users won't be prompted to upgrade to Fr10.0

Reply:
I really don't understand the upgrade mechanism of SonarQube.
Why users won't be prompted to upgrade to Fr10.0 if 10.0.0 range is `[10.0,LATEST]` ?